kafka_consumer/check.py

```diff
@@ -246,6 +246,7 @@ def _validate_consumer_groups(self, val):
 """)
 
     def _send_event(self, title, text, tags, type, aggregation_key):
+        """Emit an event to the Datadog Event Stream."""
```

Comment:
Do you mind fixing this?

I agree with your comments here though, this should be higher than normal severity.